specifyweb/frontend/js_src/lib/components/FormSliders/IntegratedRecordSelector.tsx

```diff
@@ -180,7 +180,9 @@ export function IntegratedRecordSelector({
                         }
                         onClick={(): void => {
                           focusFirstField();
-                          handleAdd();
+                          const resource =
+                            new collection.table.specifyTable.Resource();
+                          handleAdd([resource]);
```

Comment:
do we have to keep the resource as an array?

Reply:
For `IntegratedRecordSelector` and its children, I can't seem to find any case where `resources` needs to be an array. The only time it is treated as an array is here:
https://github.com/specify/specify7/blob/71ad206747904a98fac6043116e2c05331f5036c/specifyweb/frontend/js_src/lib/components/FormCells/FormTable.tsx#L125-L130
and even there it only ever has 1 element.

I'm not sure, I think converting it to a single resource instead of an array does make it less confusing. But there could be a case made for leaving it as an array to keep things consistent with other parts of the code.